README.md

```diff
@@ -404,6 +404,51 @@ the `backend` label under `matchLabels` for the metric.  The ingress annotation
 where the backend weights can be obtained can be specified through the flag
 `--skipper-backends-annotation`.
 
+## Hostname RPS collector
```

Comment:
Considering that this feature is supposed to cover what the existing Object based `request-per-second` feature is doing I would consider naming it just `request-per-second` (more generic) and allow multiple hostnames to be defined. This way it would cover a case we have in Zalando where a single application is served by multiple hostnames e.g. `app.example.com` and `app.ingress.cluster.local` (internal to the cluster). In the Object based metric we aggregate the RPS over all hostnames (see skipper collector) we could do the same here giving the users the option to define one or more hostnames.

Another feature we provide which is relevant in combination with StackSet is scaling the RPS based on traffic weight. E.g. if an application has 1000 RPS and it has two versions where one is getting 30% and the other 70% then we scale the RPS like `1000 * 30%` and `1000 * 70%` respectively, such that the stacks don't over scale during the traffic switching.
We could acheive the same if we allow passing a `scale` config (or a better name?) which is used as part of the query. The value of the `scale` config can be set by StackSet controller or manually by users if they don't use StackSet, but that use-case is less likely to be relevant.

Reply:
Ok, so the idea would be to do the same as skipper collector do, but the metric should be of type `external` and be hostname based, instead of `ingress`/`routegroup` based. Since the type - skipper is `object` and hostname is `external` - is different it should be no problem to have two collectors with the same name. Did I understood everything correctly?

Reply:
> is different it should be no problem to have two collectors with the same name. Did I understood everything correctly?

Yes, it should be no problem using the same name.

Reply:
Sounds good.

Reply:
Let's change the title here to be just `RPS Collector`?